src/js/flows/exportQueryLib.ts

```diff
@@ -0,0 +1,19 @@
+import fs from "fs"
```

Comment:
Hmm, placing a thunk directly in the api definition seems out of place to me since thunks are so coupled with redux. It seems like placing it here right next to our other thunks would at least make it easier to remember and refactor later on, no?